src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableStore.kt

```diff
@@ -114,6 +138,28 @@ class VariableStore(
     }
   }
 
+  /**
+   * Fetches the list of non-section variables, using the latest version of each based on their
+   * stable ID. Table columns are not included in the top-level list, but rather in the containing
+   * variables' lists of children.
+   */
+  fun fetchAllNonSectionVariables(): List<Variable> {
+    return with(VARIABLES) {
+      dslContext
+          .select(ID, STABLE_ID)
```

Comment:
There needs to be an aggregation function on `ID` for the `groupBy` to be valid.

Try `select(DSL.max(ID), STABLE_ID)` and remove the `orderBy`.